### ISSUE TITLE
[BUG] Conditionally rendered text in kirby-button is broken

### ISSUE THREAD

**Issue by sebastian639559** [kirbydesign/designsystem#3400]
## Describe the bug

Conditional rendering text ('render text' vs ' not rendering any text') alongside an icon inside `kirby-button` "breaks" the button. When I say it "breaks", I mean the kirby-button still behaves/'is styled' as though there is text inside the button when the conditional rendering defaults to not render any text.

## Describe how to reproduce the bug

1. Add `kirby-button` to a button html tag.
2. Inside the button, add some conditional logic that switches between rendering text and not rendering any text
3. Add an icon inside the button 

Examples:
```
            <button
              kirby-button
            >
              <kirby-icon name="cog"></kirby-icon>
              {{
                showEditOveriewLabelText$() ? "Rediger overblik" : ""
              }}
            </button>
```

```
           <button
              kirby-button
            >
              <kirby-icon name="cog"></kirby-icon>
              @if (showEditOveriewLabelText$()) {
                   "some-text"
              }
            </button>
```

## Which Kirby version was used?
    "@kirbydesign/designsystem": "9.2.1",

## What was the expected behavior?

This is how it currently behaves when the conditional logic renders an empty string:

![image](https://github.com/kirbydesign/designsystem/assets/154597334/49fd5396-e1c5-4da1-9eb5-6597f90e8807)

This is how it should behave:

![image](https://github.com/kirbydesign/designsystem/assets/154597334/cbf22562-702b-4542-abeb-5568c6ad16bb)

## Please complete the following information:
- OS: [e.g. Windows, MacOS, iOS]
- Browser [e.g. Chrome, Safari]
- Browser version [e.g. 22]

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
